deisctl/units/deis-kube-apiserver.service

```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=Kubernetes API Server
+Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+Requires=etcd2.service fleet.service docker.service flanneld.service
```

Comment:
I think we can remove the etcd2.service as flannel requires etcd and apiserver requires flannel